app/models/katello/repository.rb

```diff
@@ -183,6 +183,8 @@ class Repository < Katello::Model
     scoped_search :on => :redhat, :complete_value => { :true => true, :false => false }, :ext_method => :search_by_redhat
     scoped_search :on => :container_repository_name, :complete_value => true
     scoped_search :on => :description, :only_explicit => true
+    scoped_search :on => :name, :relation => :product, :rename => :product_name
+    scoped_search :on => :id, :relation => :product, :rename => :product_id
```

Comment:
these look fine, but you do you mind adding a test for them?  It should be fairly simple, we generally test all of our searches.